examples/RoleAssignment/RoleAssignment_Example.md

```diff
@@ -0,0 +1,50 @@
+# How To Run RoleAssignment.Tests.ps1
+
+`RoleAssignment.Tests.ps1` contains examples of using the `Confirm-AzBPRoleAssignment` cmdlet.
+
+## Pre-Requisites
+
+- Follow the [setup instructions](../README.md)
+
+## Steps
+
+1. Navigate to RoleAssignment directory:
+
+   ```Powershell
+   cd examples\RoleAssignment\
+   ```
+
+1. Deploy the Role Assignment to your subscription:
+  You will need Owner permission or higher
+
+   ```Powershell
+    New-AzSubscriptionDeployment -TemplateFile ".\roleAssignment.bicep" `
```

Comment:
This call has to provide the Service Principal ID parameter, no? If not, how is the `principalId` determined when the deployment occurs?